MonMaraicher/Features/FarmerDetail/FarmerDetailsViewModel.swift

```diff
@@ -21,11 +21,16 @@ struct FarmerDetailsViewModel: Identifiable, Hashable {
     let farmerAddressesTypes: [String]
     let city: String
 
+    var getPhoneCallURL: URL {
```

Comment:
wrong naming here. prefixing by a verb is for func not property even there are computed. `phoneCallURL` is self-sufficient